auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -840,7 +953,7 @@ private Version requiredVersion(final LoadedIdentity identityToWrite) {
             // if new functionality is used then use the required schema version otherwise fallback
             // to an older version.
 
-            if (secretKey != null) {
+            if (secretKey != null || privateKey != null) {
                 return Version.VERSION_1_1;
```

Comment:
Since version 1.1 was already been included in a release, we need to add new version 1.2 namespace.